prov/gni/src/gnix_nic.c

```diff
@@ -1127,7 +1127,7 @@ int gnix_nic_alloc(struct gnix_fid_domain *domain,
 		ret = _gnix_fl_init_ts(sizeof(struct gnix_vc),
 				       offsetof(struct gnix_vc, fr_list),
 				       GNIX_VC_FL_MIN_SIZE,
-				       0,
+				       GNIX_FL_INIT_REFILL_SIZE,
```

Comment:
Add a new #define called GNIX_VC_FL_REFILL_SIZE, since GNIX_FL_INIT_REFILL_SIZE should go away.